examples/loop_optimizations_service/service_py/utils.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+def extract_statistics_from_ir(ir: str):
```

Comment:
Seems a bit fragile? 🙂 I'm happy to merge it, but I think it may help to add a comment saying that there are caveats to this simple text approach (unless there aren't!)

Reply:
I agree. I added a comment for now. I recall my objective was to use AutoPhase, but did this as a quick start. I will create a future PR to use AutoPhase instead.